components/HowTo.tsx

```diff
@@ -0,0 +1,40 @@
+import { Flex, Heading, OrderedList, ListItem, Text } from "@chakra-ui/layout";
+
+// TODO: Copy is a placeholder and pulled from their docs
+const HowTo = () => {
+  return (
+    <Flex
+      direction="column"
+      border="1px solid"
+      borderRadius="0.3rem"
+      padding="2rem"
+      maxWidth="40rem"
+      marginTop="3rem"
+      marginBottom="3rem"
+    >
+      <Heading as="h1" size="lg" alignSelf="center">
+        Tutorial
+      </Heading>
+      <Heading as="h4" size="sm" marginTop="1rem">
+        How to shield?
```

Comment:
```suggestion
        How to shield funds?
```

Reply:
I think your understanding is correct here